### DISCUSSION TITLE
Restart after v1.3.2

### DISCUSSION THREAD

**Discussion by JHPWBN** [frenck/spook#432] [Q&A]
Hi,

I am fairly new to HA but am using Spook. I see the comment that after upgrading to v1.3.2, you need to restart using Terminal. I thought I'd check that before upgrading and tried to run "ha core restart" in the Terminal Window.

I get the error: "command not found: ha".

I appreciate some help as I do want to update Spook but am already on HA 2023.11

many thanks in advance!

Regards,
Joop

**Comment by JHPWBN** [comment]
I just realised that I am running HA on HA Yellow so running Terminal on my iMac does not help I guess :-)
HA Yellow is a complete mystery for me on Operating System level, don't even know how to get into the operating system, let alone to restart HA from there :-(

Much help appreciated !!!

**Reply by frenck** [reply]
It refers to one of the web terminals inside HA itself (provided by one of the SSH add-ons).


Alternatively, you could also do a force restart by going into the developer tools, service tab. Search for the "homesssistant.restart" service, check the force restart checkbox and call the service.

**Reply by JHPWBN** [reply]
Thanks a million! That is how simple it gets when you ask the right people!
